### PR TITLE
[FLINK-6330] [docs] Add basic Docker, K8s docs

### PR DESCRIPTION
**Pull request by patricklucas**
I may do some more work on these before 1.2.2/1.3, but I'd like to at least get this basic information into master.

### CODE REVIEW

**Comment by StephanEwen** [comment]
Great, thanks!

Merging this...

**Comment by StephanEwen** [comment]
Sorry, taking a step back.

It seems that https://hub.docker.com/_/flink/ is not yet available. Should we postpone this merge until it is available?

**Comment by patricklucas** [comment]
It will be available before the next release, though I noticed just yesterday that it seems we build the [official 1.2 docs](https://ci.apache.org/projects/flink/flink-docs-release-1.2/) (erroneously?) from a development branch instead of the actual tagged release.

I'm about ready to submit the PR to get the official images created, but it's fine if you want to wait.

**Comment by greghogan** [comment]
@patricklucas when will the link be available? It would be great to have this merged for 1.3.

**Comment by patricklucas** [comment]
@greghogan we're waiting patiently for @tianon to complete his review of docker-library/official-images#2895. :)

As far as I know, we've done everything we need to do to get the images published. We have a +1 from @yosifkit, but two reviews are required.

**Comment by patricklucas** [comment]
The link is available now!

File: docs/page/css/syntax.css [hunk 1]
```diff
@@ -77,3 +77,6 @@
 .highlight .vg { color: #008080 } /* Name.Variable.Global */
 .highlight .vi { color: #008080 } /* Name.Variable.Instance */
 .highlight .il { color: #009999 } /* Literal.Number.Integer.Long */
+
+/* Missing YAML styles */
+.highlight .l-Scalar-Plain { color: #008080 }
```
Comment (greghogan) [inline docs/page/css/syntax.css]: What is this line doing? Why does `YAML` equate to `l-Scalar-Plain`? Is `YAML` unknown to the highlighter?

Comment (patricklucas) [inline docs/page/css/syntax.css]: Yeah, the highlighter we use doesn't have any built-in styling for YAML. I had to search around quite a bit for context and came up mostly dry.

Moreover, the actual tokenizer it uses only emits two token types, l-Scalar-Plain and p-Indicator. (You can see this in action by inspecting the HTML [here](https://ci.apache.org/projects/flink/flink-docs-release-1.2/monitoring/metrics.html#jmx-orgapacheflinkmetricsjmxjmxreporter))

This change applies the color I found most applicable from the existing highlighting rules and applies it to l-Scalar-Plain to make it at least slightly more interesting than black-on-white.

Comment (greghogan) [inline docs/page/css/syntax.css]: I think we should merge this PR without this change and look at fixing this in another ticket. There are [other](https://highlightjs.org) highlighters with true YAML support.

**Pull request closed without merging**